package.json

```diff
@@ -267,24 +268,24 @@
         "immutable": "^3.7.4",
         "jsuri": "^1.3.1",
         "lodash": "^4.17.5",
-        "mousetrap": "^1.6.1",
-        "pikaday": "^1.6.1",
+        "mousetrap": "^1.6.3",
```

Comment:
Should we be making peers use newer patches?

Reply:
Looks like I missed a few. Will push an updated list.